posthog/src/main/java/com/posthog/internal/GzipRequestInterceptor.kt

```diff
@@ -48,18 +48,18 @@ internal class GzipRequestInterceptor(private val config: PostHogConfig) : Inter
         ) {
             chain.proceed(originalRequest)
         } else {
-            val compressedRequest =
-                try {
-                    originalRequest.newBuilder()
-                        .header("Content-Encoding", "gzip")
-                        .method(originalRequest.method, forceContentLength(gzip(body)))
-                        .build()
-                } catch (e: Throwable) {
-                    config.logger.log("Failed to gzip the request body: $e.")
-
-                    originalRequest
-                }
-            chain.proceed(compressedRequest)
+//            val compressedRequest =
+//                try {
+//                    originalRequest.newBuilder()
+//                        .header("Content-Encoding", "gzip")
+//                        .method(originalRequest.method, forceContentLength(gzip(body)))
+//                        .build()
+//                } catch (e: Throwable) {
+//                    config.logger.log("Failed to gzip the request body: $e.")
+//
+//                    originalRequest
+//                }
```

Comment:
Android errors with gson using flags but not with /decide
https://posthog.slack.com/archives/C07Q2U4BH4L/p1749803054185859